.gitignore

```diff
@@ -1,3 +1,2 @@
 /vendor
-composer.lock
-composer.phar
\ No newline at end of file
+composer.phar
```

Comment:
I prefere to not having it simply because it should always work with latest dependencies specified in composer.json !